adr/0008-named-callbacks.md

```diff
@@ -0,0 +1,55 @@
+# 7: LogAs Function in Capability Class "Named Callbacks"
```

Comment:
"LogAs" seems kinda limited to me. :thinking: 

We talked about this being a general purpose feature that could / would (essentially) carry a label for any given callback around throughout the Pepr system, right?  As it stands, the only place we're currently showing callback function names is in the log messages but that's not all we think can / will be done with this IMO, and I'd like to try to avoid handicapping ourselves by giving it an overly limiting name.

What would you say to naming this something that hinted at a more general usage?  Something like:
- Alias()
- Designation()
- Handle()
- or maybe simply, As()

All of those give a good sense of what we want this new func to be thought of as _without_ necessarily limiting it to logs.

Reply:
Interesting prospect here, not opposed by any means. Where else other than logs do you see this name being used?

Reply:
I think we like Alias because a user who has not read the docs is most likely to interpret this as a name.